python_modules/dagster/dagster/_core/definitions/decorators/asset_decorator.py

```diff
@@ -208,6 +220,7 @@ def __init__(
         op_tags: Optional[Dict[str, Any]] = None,
         group_name: Optional[str] = None,
         output_required: bool = True,
+        version: Union[bool, str] = False,
```

Comment:
What do you think about calling this "op_version" to make it clear that it's the version of the computation, not the version of the asset?